aphrodite/common/sampling_params.py

```diff
@@ -400,6 +402,9 @@ def _verify_args(self) -> None:
         if self.repetition_penalty < 1.0:
             raise ValueError("repetition_penalty must be in [1, inf), got "
                              f"{self.repetition_penalty}.")
+        if self.rep_range is not None and self.rep_range < 1:
```

Comment:
We should probably allow 0 for infinite (or all tokens) range. Unless other inference software do it this way.

Reply:
You're absolutely right, I'll implement on Monday